miden/src/cli/bundle.rs

```diff
@@ -40,19 +39,19 @@ impl BundleCmd {
                 .into_owned(),
         };
 
-        let source_manager = Arc::new(assembly::DefaultSourceManager::default());
+        let assembler = Assembler::default().with_debug_mode(true);
         let library_namespace =
             namespace.parse::<LibraryNamespace>().expect("invalid base namespace");
         // TODO: Add version to `Library`
-        let _version = self.version.parse::<Version>().expect("invalid cargo version");
-        let stdlib = Library::from_dir(&self.dir, library_namespace, source_manager)?;
+        // let version = self.version.parse::<Version>().expect("invalid cargo version");
```

Comment:
Similarly I would remove the commented out code

Reply:
Agreed! Removed.